Tests/Schema.NET.Test/ValuesTest.cs

```diff
@@ -0,0 +1,57 @@
+namespace Schema.NET.Test
+{
+    using System;
+    using Newtonsoft.Json;
+    using Xunit;
+
+    public class ValuesTest
+    {
+        [Fact]
+        public void CanSetValuesIntUsingString()
+        {
+            Values<int> vInt = (Values<int>)"someTest";
```

Comment:
I'm not sure I get this yet but why accept `string` for every type?

Reply:
I actually removed that ;)

But I am having some issues with parsing data. I had some sites running a brand, and it was specified as brand="XXXXX". Brand can not have string, needs to be strong type Brand or Organization. Also I am wondering about author="XXX" for creativework.

It didn't feel ok, so I removed it again. I do need to think about types string to brand, string to author etc. But I don't think that should be done in the schema.org classes. If there are any ideas on how to handle those hard cases where string is not allowed, I actually don't want to loose data, because it doesn't fit the objects.

At this moment I am more leanings against an extra Dictionary in Thing. Containing all stuff that doesn't fit where it is supposed to fit. Don't know about that one either ..... schema.org is too big to manually work out all string -> object conversions (at least I think so)